Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -77,7 +80,7 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, b
 
 if ~xASL_exist(PWI_Path, 'file')
     warning('Skipped xASL_wrp_Quantify: files missing, please rerun step 4: xASL_wrp_ResampleASL');
-	if bUseBasilQuantification
+	if x.Q.bUseBasilQuantification
```

Comment:
At the header it is without `x.Q`, see `xASL_module_ASL` that calls this wrapper.

Reply:
Yes - you need to be very careful to distinguish between local variables and global ones saved in x-struct. Especially since you now return the x-struct.

Reply:
Ah damn, okay, so the idea was to make it possible to run some sessions with and some without BASIL? Didn't knew that this was a use-case.